src/core/middleware/session.extend.middleware.ts

```diff
@@ -58,9 +58,7 @@ export class SessionExtendMiddleware implements NestMiddleware {
 
     let session: Session;
     try {
-      session = await getSession(this.kratosClient, {
-        authorization,
-      });
+      session = getSessionFromJwt(authorization);
```

Comment:
this function requires the token, and you are passing the whole value of the header.
`getSession` was able to parse the header, but this one doesn't

Reply:
Turns out that it actually can parse the whole header.